gbfs-validator/__test__/gbfs.v3.0-RC.test.js

```diff
@@ -137,8 +155,12 @@ describe('exaustive feed', () => {
 
     expect.assertions(2)
 
-    return gbfs.validation().then(result => {
-      expect(get_errors(result)).toEqual([])
+    return gbfs.validation().then((result) => {
+      expect(get_errors(result)).toEqual({
+        errors: [],
+        nonSchemaErrors: [],
+        warnings: []
```

Comment:
The taxonomy here feels odd. There are errors and there are warnings. But there are also a specific kind of error, which is not from the schema. Where do warnings come from?

My hope is that validation results become standardised for portability reasons, so I hope we can spend the extra effort to consider a rational api.